src/DoctrineORMModule/Service/DBALConfigurationFactory.php

```diff
@@ -9,60 +11,55 @@
 use Laminas\ServiceManager\FactoryInterface;
 use Laminas\ServiceManager\ServiceLocatorInterface;
 use RuntimeException;
+use function is_string;
+use function sprintf;
 
 /**
  * DBAL Configuration ServiceManager factory
  */
 class DBALConfigurationFactory implements FactoryInterface
 {
-    /**
-     * @var string
-     */
+    /** @var string */
     protected $name;
 
-    /**
-     * @param string $name
-     */
-    public function __construct($name)
+    public function __construct(string $name)
     {
         $this->name = $name;
     }
 
     /**
      * {@inheritDoc}
      *
-     * @return \Doctrine\DBAL\Configuration
+     * @return Configuration
      */
-    public function __invoke(ContainerInterface $container, $requestedName, array $options = null)
+    public function __invoke(ContainerInterface $container, $requestedName, ?array $options = null)
```

Comment:
I think it would make sense to use a return type declaration here and below, it's still compatible with the parent method, isn't it?

Reply:
And above too, in all factories in fact.

Reply:
This factory pattern is what the parent classes use too so I'm not inclined to try to improve it.